src/iohumdrum.cpp

```diff
@@ -756,7 +841,7 @@ void HumdrumInput::insertTitle(pugi::xml_node &titleStmt, const vector<HumdrumLi
         plang = false;
         lang = false;
         key = references[i]->getReferenceKey();
-        if (key.compare(0, 3, "OTL") != 0) {
+        if (key.compare(0, 2, "OT") && key.compare(0, 1, "X")) {
```

Comment:
This condition is cryptic, but studying it for a while it makes sense.  It means: don't process keys starting with neither "OT" or "X" (compare returns false if they match which was the confusing part).  Keys starting with "OT" or "X" are all title-related ("X" ones are in an older style which I replaced with the `@`/`@@` language modifiers which affect any reference key).